### DISCUSSION TITLE
批量生效参数调整（参数是通过sql修改的item表）

### DISCUSSION THREAD

**Discussion by BruceMiao305** [apolloconfig/apollo#4056] [Q&A]
有时候配置的IP参数发生变化时，并且这个参数又被多个项目使用的时候，我看可以在数据库中批量替换value值，但是每次都得手工去每个项目下面发布才能生效，有没有更快的方式批量生效的？

**Comment by hhh220** [comment]
把公共配置都写到一个公共namespace里面，配置不同的就关联它并替换value，这样就不用重新发布项目了哈

**Comment by BruceMiao305** [comment]
这种方式肯定不行的，有些ftp或者数据库的jdbc配置都是写了schema库的，是不一样的配置，我主要是咨询的是从数据库层面改配置，如何不需要去页面发布生效，有没有方式能在数据库中使其生效的方式。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "apolloconfig/apollo"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年10月21日(星期四) 下午4:35
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [apolloconfig/apollo] 批量生效参数调整（参数是通过sql修改的item表） (Discussion #4056)





 
把公共配置都写到一个公共namespace里面，配置不同的就关联它并替换value，这样就不用重新发布项目了哈
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
